### ISSUE TITLE
Move Accepted Payment Method to Another Tab Under Checkout

### ISSUE THREAD

**Issue by BFTrick** [jameskoster/woocommerce-accepted-payment-methods#15]
Hey James,

What do you think about moving all of this to a separate tab under checkout?

Something like this:
![woocommerce_settings_ _woocommerce_ _wordpress](https://cloud.githubusercontent.com/assets/1065372/5767871/f162b3ac-9cc8-11e4-9865-27dedb24ee5f.png)

**Comment by jameskoster** [comment]
@BFTrick yeah all of my plugins could do with an update. The admin has changed a lot since I released them. Just gotta find the time!
